app/Console/Commands/DeactivateDonatorPerksCommand.php

```diff
@@ -80,9 +86,17 @@ public function handle()
                         $this->syncAction->syncAll();
 
                         $expiredPerk->account->groups()->detach($donatorGroup->getKey());
+
+                        $expiredPerk->account->load("groups");
```

Comment:
Just a tiny nit for consistency
```suggestion
                        $expiredPerk->account->load('groups');
```

Reply:
Though I feel like we're doing something to wrong to need to call `load` after every group modification 🤔

Reply:
It's a weird one but I run into this all the time. I believe it's just a side effect of the way attach and detach works with respect to the memory representation of models, where it doesn't re-query the DB to see what relationships were actually created for performance reason.